store-implementation/proxy-store/src/main/java/uk/gov/gchq/gaffer/proxystore/ProxyProperties.java

```diff
@@ -37,7 +37,7 @@ public class ProxyProperties extends StoreProperties {
     public static final String READ_TIMEOUT = "gaffer.read-timeout";
 
     public static final String DEFAULT_GAFFER_HOST = "localhost";
-    public static final String DEFAULT_GAFFER_CONTEXT_ROOT = "/rest/v2";
+    public static final String DEFAULT_GAFFER_CONTEXT_ROOT = "/rest";
```

Comment:
What was the reasoning behind this change?

Reply:
Spring doesn't have V2 by default.

Reply:
We should document that proxy by default now wants to point to a spring-rest and not the older one